tests/test_execute.py

```diff
@@ -38,7 +38,7 @@ def doctree(source, config=None, return_warnings=False, entrypoint="jupyter_sphi
         apps.append(app)
         app.build()
 
-        doctree = app.env.get_doctree("contents")
+        doctree = app.env.get_and_resolve_doctree("contents", app.builder)
```

Comment:
@choldgraf following a sphinx code dive I figured that this should also apply the post-transforms.

@chrisjsewell does this make sense?

Reply:
do you mean calling this function also applies post-transforms?

Reply:
Yep. I verified that after this modification of the fixture, the produces doctrees are correct.

Reply:
huh, interesting...so, does that mean we need to change something or that it's OK as-is?

Reply:
The failures that I see now are due to the AST change (there's now `CellInputNode` containing a literal block and not just a literal block), and not due to the post-transform not being applied.

Reply:
...so unless I'm overlooking something, this should be good.